lib/middleware/validateSession.js

```diff
@@ -0,0 +1,70 @@
+var getSessionVal = function(obj, key) {
```

Comment:
This function can be removed. It's not worth  the complexity as the raincatcher-user module only works in the context of the RMAP Platform anyway.

We can make the assumption that the user is using a cloud app.